src/Resources/views/Form/datepicker.html.twig

```diff
@@ -67,3 +85,21 @@ file that was distributed with this source code.
         </script>
     {% endspaceless %}
 {% endblock sonata_type_datetime_picker_widget %}
+
+{% block sonata_type_datetime_range_picker_widget %}
+    {% spaceless %}
+    {{ block('form_widget') }}
+        <script type="text/javascript">
```

Comment:
This block looks like a copy paste from the other one, extract it and reuse. Are you sure we dont have this block somewhere already?

Reply:
all block in this file are duplicated from date_picker and date_time picker just for one char in the id: dtp_xxx vs dp_xxx.
I wanted to keep it in the same style, but you're right

Reply:
Bonus points for a separate js files that just parses the DOM and extracts needed data from data-attributes (no js in Twig would be great)

Reply:
All the form type in this bundle have the js embedded in the twig.
If we want to externalize it, it as to be called on page load and by Admin.shared_setup (on modal open) in the AdminBundle.
I could maybe simple share a twig block.